File: Anul II/Medii de proiectare si programare/Laborator/lab06/networking/src/main/java/com/ubb/protocol/response/UpdatedRacesResponse.java
package com.ubb.protocol.response;

import com.ubb.dto.RaceDTO;

import java.util.List;

public class UpdatedRacesResponse implements UpdateResponse {
    private final List<RaceDTO> races;

    public UpdatedRacesResponse(List<RaceDTO> races) {
        this.races = races;
    }

    public List<RaceDTO> getRaces() {
        return races;
    }
}

File: Anul II/Medii de proiectare si programare/Laborator/lab07/Ubb.BikeContest.Networking/Response/UpdatedRacesResponse.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Ubb.BikeContest.Model.data;

namespace Ubb.BikeContest.Networking.Response
{
    [Serializable]
    public class UpdatedRacesResponse : UpdateResponse
    {
        private readonly List<RaceDto> races;

        public UpdatedRacesResponse(List<RaceDto> races)
        {
            this.races = races;
        }

        public List<RaceDto> Races { get { return races; } }
    }
}
